File: Dante1/Module1.05/Zadanie5.01/main.c

#include <stdio.h>

int main()
{
    int data[26];

    int number = 65;
    for(int i = 0; i < 26; i++)
    {
        data[i] = number;
        number++;
    }
    for(int i = 0; i < 26; i++)
    {
        printf("%d %c %c\n",data[i],data[i],data[i] + 32);
    }
    return 0;
}

File: Dante1/Module1.05/Zadanie5.01/main.rs
fn main()
{
    let mut data:[u8; 26] = [0; 26];
    let mut number: u8 = 65;
    
    for i in 0..26
    {
        data[i] = number;
        number += 1;
    }
    
    for i in 0..26
    {
        println!("{} {} {}", data[i], data[i] as char, (data[i] + 32) as char);
    }
}
